packages/cohort_banking/src/callbacks/statemap_installer.rs

```diff
@@ -0,0 +1,83 @@
+use std::sync::Arc;
+
+use async_trait::async_trait;
+use cohort_sdk::{model::callbacks::StatemapInstaller, replicator::core::StatemapItem};
+use tokio_postgres::types::ToSql;
+
+use crate::{model::requests::TransferRequest, state::postgres::database::Database};
+
+pub struct StatemapInstallerImpl {
+    pub database: Arc<Database>,
+}
+
+#[async_trait]
+impl StatemapInstaller for StatemapInstallerImpl {
+    async fn install(&self, statemap: Vec<StatemapItem>, snapshot_version: u64) -> Result<(), String> {
+        // from      = 1
+        // to        = 2
+        // amount    = 3
+        // new_ver   = 4
+
+        let mut cnn = self.database.get().await.map_err(|e| e.to_string())?;
+        // let tx = cnn.transaction().await.map_err(|e| e.to_string())?;
+        let tx = cnn
+            .build_transaction()
+            .isolation_level(tokio_postgres::IsolationLevel::Serializable)
```

Comment:
Isolation level should be `RepeatableRead` ?

Reply:
Yes, thanks.